### ISSUE TITLE
Update NGPVAN.apply_canvass_result to include Phone

### ISSUE THREAD

**Issue by elyse-weiss** [move-coop/parsons#525]
Parsons link: https://move-coop.github.io/parsons/html/ngpvan.html?highlight=canvass#parsons.ngpvan.van.People.apply_canvass_result

NGPVAN documentation: https://docs.ngpvan.com/reference/people?emci=3c487bf6-03b7-eb11-a7ad-0050f271b5d8&emdi=f24ba893-acb8-eb11-a7ad-501ac57ba3ed&ceid=6426064#peoplevanidcanvassresponses

NGPVAN sent an email that they will start requiring phone for applying canvass results with that canvass type.

**Comment by mkwoods927** [comment]
Update: [phone is now required](https://docs.ngpvan.com/changelog/phone-number-is-now-required-for-certain-canvass-responses) for input types Phone and SMS!

**Comment by bzupnick** [comment]
What's the current status of this? I'm happy to take this, but does it make sense that this function hasn't worked for over a year?

Also, how do I test NGPvan functions?

**Comment by bzupnick** [comment]
It turns out that [the changelog](https://docs.ngpvan.com/changelog/phone-number-is-now-required-for-certain-canvass-responses) has an error in it, which led to some confusion on my part, but I was able to reproduce the issue and a fix is en-route.

The changelog says `when canvassContext.inputTypeId is 1 (Phone) or 37 (SMS)`. where as it SHOULD have said `when canvassContext.contactTypeId is 1 (Phone) or 37 (SMS)`.
Note the `inputTypeId` --> `contactTypeId`.

I confirmed this by hitting `get_canvass_responses_contact_types` and I see them! I see `contactTypeId`'s `1` and `37` connoting phone and SMS
